macros/src/lib.rs

```diff
@@ -11,12 +11,12 @@
 #![crate_type="dylib"]
 
 #![feature(plugin_registrar, quote)]
-#![feature(rustc_private, core, std_misc)]
+#![feature(rustc_private, convert)]
```

Comment:
@Ogeon, I don't think the issue is necessarily with `#![feature]`. I've been hitting a similar bug, without using `#![feature]` at all and on older version of Rust. I suspect it's some underlying macro expansion error.